### PR TITLE
Replaced default algorithm implementations with FIPS-compliant versions.

### PR DESCRIPTION
**Pull request by andresjacobo**
Please let me know if anything else needs to be done. Thanks!

### CODE REVIEW

File: UnitTests/project.json [hunk 1]
```diff
@@ -26,10 +26,11 @@
           }
         }
     },
-    "dependencies": {
-        "jose-jwt": "2.0.0",
-        "xunit": "2.1.0"
-    },
+  "dependencies": {
+    "jose-jwt": "2.0.0",
+    "Microsoft.NETCore.App": "1.1.0",
```
Comment (dvsekhvalnov) [inline UnitTests/project.json]: mm...understood it is tests, but why do we need this reference to Microsofr.NETCore.App ?

**Comment by dvsekhvalnov** [comment]
looks good to me, just see one small comment.

File: UnitTests/project.json (hunk 1 above)
Comment (andresjacobo) [inline UnitTests/project.json]: Hmmmm.... I'm sorry, I didn't know these files got changed. You should be able to ignore them if you wish. Really the only changes are in AesKeyWrap.cs and RsaUsingSha.cs

**Comment by dvsekhvalnov** [comment]
Will try to push by end of week to nuget. Have one incompatible change in master, need to revert.

**Pull request merged**